__tests__/getUsers.test.js

```diff
@@ -0,0 +1,43 @@
+const app = require("../app");
+const seed = require("../db/seeds/seed");
+const db = require("../db/connection");
+const request = require("supertest");
+const testData = require("../db/data/test-data/index.js");
+
```

Comment:
Having loads of different test files is unnecessary - you can put all your app tests in one.